policies/expires_after_tagging/policy.json

```
@@ -0,0 +1,30 @@
+{
+    "properties": {
+      "displayName": "Expires After Tagging",
```

most of our policies have hmcts at the start prob good to make it easy to search
```suggestion
      "displayName": "HMCTS Expires After Tagging",
```